[2m2025-03-06T16:24:06.882918Z[0m [32m INFO[0m [2medamame_core::core_manager[0m[2m:[0m Initializing core with language: EN, device_id: 005EF6A9-1F73-5963-9DD4-7385F346CCCE, model: , brand: , os_name: macOS, os_version: macOS 15.3.1, branch: main, computing_enabled: true, reporting_enabled: false, community_enabled: false, server_enabled: false, analytics_enabled: true
[2m2025-03-06T16:24:06.883904Z[0m [32m INFO[0m [2medamame_core::core_manager[0m[2m:[0m Launched in standalone mode
[2m2025-03-06T16:24:06.885476Z[0m [32m INFO[0m [2medamame_core::core_platform[0m[2m:[0m Platform '' initialized with semver '15.3.1' and os name 'macOS'
[2m2025-03-06T16:24:06.886116Z[0m [32m INFO[0m [2medamame_core::core_manager[0m[2m:[0m Running in a non-CI/CD environment
[2m2025-03-06T16:24:06.886749Z[0m [32m INFO[0m [2medamame_core::core_communications[0m[2m:[0m Language set to 'EN'
[2m2025-03-06T16:24:06.887366Z[0m [32m INFO[0m [2medamame_core::core_communications[0m[2m:[0m Community is disabled
[2m2025-03-06T16:24:06.901463Z[0m [32m INFO[0m [2medamame_core::core_manager[0m[2m:[0m Connection status: ConnectionStatus { connected_user: "", connected_domain: "", pin: "", is_success: true, is_connected: false, is_success_pin: false, is_outdated: false, last_network_activity: "2025-03-06T16:23:47.113465+00:00", error_code: None, error_reason: "", last_report_time: Some(2025-03-06T16:23:47.113451Z), last_report_signature: "P+vSeGApD6MFntU9aj5gz6eVPsjjLRI3FDOMWWZVmco=" }
[2m2025-03-06T16:24:06.902105Z[0m [32m INFO[0m [2medamame_core::core_manager[0m[2m:[0m Not connected to domain
[2m2025-03-06T16:24:06.903777Z[0m [32m INFO[0m [2medamame_core::core_timer[0m[2m:[0m Starting timer 'compute_score_task'
[2m2025-03-06T16:24:06.904787Z[0m [32m INFO[0m [2medamame_core::core_timer[0m[2m:[0m Starting timer 'update_score_task'
[2m2025-03-06T16:24:06.905649Z[0m [32m INFO[0m [2medamame_core::core_manager_tasks[0m[2m:[0m compute_score_task called
[2m2025-03-06T16:24:06.905662Z[0m [32m INFO[0m [2medamame_core::core_timer[0m[2m:[0m Starting timer 'app_watcher_task'
[2m2025-03-06T16:24:06.906444Z[0m [32m INFO[0m [2medamame_core::core_timer[0m[2m:[0m Starting timer 'update_task'
[2m2025-03-06T16:24:06.908553Z[0m [32m INFO[0m [2medamame_core::user_email[0m[2m:[0m User email from json: frank@lyonnet.org
[2m2025-03-06T16:24:06.909616Z[0m [32m INFO[0m [2medamame_core::core_timer[0m[2m:[0m Starting timer 'lanscan_loop_task'
[2m2025-03-06T16:24:06.909745Z[0m [32m INFO[0m [2medamame_core::core_timer[0m[2m:[0m Deferring start of task 'update_task' by 300 seconds
[2m2025-03-06T16:24:06.910746Z[0m [32m INFO[0m [2medamame_foundation::threat_factory[0m[2m:[0m Loading threat metrics from JSON for platform: macos
[2m2025-03-06T16:24:06.911325Z[0m [32m INFO[0m [2medamame_core::core_state[0m[2m:[0m Computing pwned state for email frank@lyonnet.org
[2m2025-03-06T16:24:06.912323Z[0m [32m INFO[0m [2medamame_core::core_timer[0m[2m:[0m Deferring start of task 'lanscan_loop_task' by 60 seconds
[2m2025-03-06T16:24:06.912705Z[0m [32m INFO[0m [2medamame_foundation::threat_factory[0m[2m:[0m Loaded 27 threat metrics
[2m2025-03-06T16:24:06.912782Z[0m [32m INFO[0m [2medamame_core::pwned_factory[0m[2m:[0m Initializing breach cache with email: frank@lyonnet.org
[2m2025-03-06T16:24:06.913994Z[0m [32m INFO[0m [2medamame_core::pwned_factory[0m[2m:[0m Loaded 18 breaches from cache pwned_19ed9b871a5771ea1325155a1fa155afb53d60c88facb6ffb44a1f3f002ab6b3
[2m2025-03-06T16:24:06.914623Z[0m [32m INFO[0m [2medamame_core::pwned_factory[0m[2m:[0m Breach cache is still valid
[2m2025-03-06T16:24:06.915303Z[0m [32m INFO[0m [2medamame_core::core_state[0m[2m:[0m Retrieved 18 breaches
[2m2025-03-06T16:24:06.915930Z[0m [32m INFO[0m [2medamame_core::core_state[0m[2m:[0m Pwned state potential is true
[2m2025-03-06T16:24:06.917610Z[0m [32m INFO[0m [2medamame_core::lanscan_factory[0m[2m:[0m No interfaces, skipping scan...
[2m2025-03-06T16:24:06.918201Z[0m [32m INFO[0m [2medamame_core::score_factory[0m[2m:[0m Starting a new computation
[2m2025-03-06T16:24:06.921858Z[0m [32m INFO[0m [2medamame_core::remediation_cache[0m[2m:[0m Loaded 18 cache entries from path pwned_remediations_cache
[2m2025-03-06T16:24:06.921911Z[0m [32m INFO[0m [2medamame_core::score_factory[0m[2m:[0m Score computation thread started
[2m2025-03-06T16:24:06.922510Z[0m [32m INFO[0m [2medamame_core::pwned_remediations[0m[2m:[0m No breach remediation to prefetch
[2m2025-03-06T16:24:07.084497Z[0m [32m INFO[0m [2medamame_core::core_manager[0m[2m:[0m Core initialized
[2m2025-03-06T16:24:07.223706Z[0m [32m INFO[0m [2medamame_core::score_factory[0m[2m:[0m Waiting for score computation to complete for the first time
[2m2025-03-06T16:24:07.574573Z[0m [32m INFO[0m [2medamame_core::runner_internal[0m[2m:[0m Internal pwned state - pwned is true and last pwned is Some(2024-09-25T00:00:00Z)
[2m2025-03-06T16:24:07.575194Z[0m [32m INFO[0m [2medamame_core::runner_internal[0m[2m:[0m Pwned before limit ( 162 < 365 days ago )
[2m2025-03-06T16:24:07.644087Z[0m [32m INFO[0m [2medamame_core::score_factory[0m[2m:[0m Score computation completed, processing results
[2m2025-03-06T16:24:07.644898Z[0m [32m INFO[0m [2medamame_core::score_factory[0m[2m:[0m Computed score: 3.7 (application: 71, network: 57, system services: 73, system integrity: 83, credentials: 58)
[2m2025-03-06T16:24:07.645659Z[0m [32m INFO[0m [2medamame_core::core_manager_tasks[0m[2m:[0m Myself updated with stars: 3.7, has_helper: false
[2m2025-03-06T16:24:08.228953Z[0m [32m INFO[0m [2medamame_core::core_manager_tasks[0m[2m:[0m Reporting score with 3.7 stars
[2m2025-03-06T16:24:08.359631Z[0m [32m INFO[0m [2medamame_core::core_communications[0m[2m:[0m Reporting score for device 005EF6A9-1F73-5963-9DD4-7385F346CCCE (macOS macOS 15.3.1) with IPv4 82.64.124.164 IPv6 2a01:e0a:17f:a660:a171:b5b7:32c9:8024 MAC D2:29:07:D5:CA:52
[2m2025-03-06T16:24:08.574287Z[0m [32m INFO[0m [2medamame_core::core_communications[0m[2m:[0m Coordinate for 82.64.124.164 - city: Antony / region: ÃŽle-de-France / country: France / timezone: Europe/Paris / lat: 48.7533 / long: 2.29668
[2m2025-03-06T16:24:08.576413Z[0m [33m WARN[0m [2medamame_core::backend_aws[0m[2m:[0m Using EDAMAME_SECURITY_LAMBDA override: https://l3xbkgey7a.execute-api.eu-west-1.amazonaws.com/e2e-tests-quentin-s-stack
[2m2025-03-06T16:24:08.578738Z[0m [32m INFO[0m [2medamame_core::backend_aws[0m[2m:[0m Executing backend request to base_url https://l3xbkgey7a.execute-api.eu-west-1.amazonaws.com/e2e-tests-quentin-s-stackreport/detail
[2m2025-03-06T16:24:08.708785Z[0m [31mERROR[0m [2medamame_core::backend_aws[0m[2m:[0m Unhandled response code 403 for url https://l3xbkgey7a.execute-api.eu-west-1.amazonaws.com/e2e-tests-quentin-s-stackreport/detail
[2m2025-03-06T16:24:08.710519Z[0m [33m WARN[0m [2medamame_core::backend_aws[0m[2m:[0m [3merror_message[0m[2m=[0m"Request report_score failed with code 403 and response body: \"{\\\"message\\\":\\\"Forbidden\\\"}\", error code is Unknown, error reason is \"\", url is https://l3xbkgey7a.execute-api.eu-west-1.amazonaws.com/e2e-tests-quentin-s-stackreport/detail"
[2m2025-03-06T16:24:08.712828Z[0m [31mERROR[0m [2medamame_core::core_connection_status[0m[2m:[0m Success flag switched to false
[2m2025-03-06T16:24:08.714725Z[0m [31mERROR[0m [2medamame_core::core_manager_tasks[0m[2m:[0m Error during score reporting: Call to backend failed with error code: Unknown
[2m2025-03-06T16:24:08.716219Z[0m [32m INFO[0m [2medamame_core::api::api_trust[0m[2m:[0m get_connection_async returned ConnectionStatus { connected_user: "", connected_domain: "", pin: "", is_success: false, is_connected: false, is_success_pin: false, is_outdated: false, last_network_activity: "2025-03-06T16:24:08.712891+00:00", error_code: Some(Unknown), error_reason: "Unknown", last_report_time: Some(2025-03-06T16:23:47.113451Z), last_report_signature: "P+vSeGApD6MFntU9aj5gz6eVPsjjLRI3FDOMWWZVmco=" }
[2m2025-03-06T16:24:08.717075Z[0m [32m INFO[0m [2medamame_core::core_manager[0m[2m:[0m Terminating core
[2m2025-03-06T16:24:08.881275Z[0m [32m INFO[0m [2medamame_core::core_manager[0m[2m:[0m Core terminated
